tests/test_utils/tests.py

```diff
@@ -914,3 +914,21 @@ def test_override_staticfiles_dirs(self):
         with self.settings(STATICFILES_DIRS=[test_path]):
             finder = get_finder('django.contrib.staticfiles.finders.FileSystemFinder')
             self.assertIn(expected_location, finder.locations)
+
+
+class DisallowedDatabaseQueriesTests(SimpleTestCase):
+    def runTest(self):
```

Comment:
Just curious if there is a reason for the `runTest` name instead of the usual non-camel case name.